_specifications/lsp/3.18/types/textEdit.md

```diff
@@ -79,3 +81,31 @@ export interface AnnotatedTextEdit extends TextEdit {
 	annotationId: ChangeAnnotationIdentifier;
 }
 ```
+
+Since 3.18.0, there is also the concept of a snippet text edit, which supports inserting a snippet instead of plain text.
```

Comment:
Shouldn't this have some client capability associated, so servers know if they can make use of it?

Reply:
There is one ion the proposed implementation so it should definitely be in the spec.

Reply:
Absolutely, the reason this is a draft PR is because it's incomplete :) I just published it to ask about the `StringValue` thing.